docs/features/traffic-management/istio.md

```diff
@@ -246,6 +246,70 @@ During the lifecycle of a Rollout using Istio DestinationRule, Argo Rollouts wil
   label of the canary and stable ReplicaSets
 
 
+## Multicluster Setup
+If you have [Istio multicluster setup](https://istio.io/latest/docs/setup/install/multicluster/)
+where the primary Istio cluster is different than the cluster where the Argo Rollout controller
+is running, then you need to do the following setup:
+
+1. Create a `ServiceAccount` in the Istio primary cluster.
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: argo-rollouts-istio-primary
+  namespace: <any-namespace-preferrably-config-namespace>
+```
+2. Create a `ClusterRole` that provides access to Rollout controller in the Istio primary cluster.
```

Comment:
Good point. If Argo controller is also installed in the primary cluster (not mandatory), then we could just use the default `argo-rollouts-clusterrole` instead of creating another in the primary cluster. I will add a note to mention the same. Thanks.